webhelper/packages.txt

```diff
@@ -0,0 +1,38 @@
+alsa-lib libasound2
+atk1.0 libatk1.0-0
+at-spi2-atk libatk-bridge2.0-0
+at-spi2-core libatspi2.0-0
+cairo libcairo2
+libdatrie libdatrie1
+dbus libdbus-1-3
+expat libexpat1
+libffi libffi6
+fontconfig fontconfig libfontconfig1
```

Comment:
@TTimo: I copied the package list from your `jessie` branch. Do we really want `fontconfig`? That package contains:

```
/usr/bin/fc-cache
/usr/bin/fc-cat
/usr/bin/fc-conflist
/usr/bin/fc-list
/usr/bin/fc-match
/usr/bin/fc-pattern
/usr/bin/fc-query
/usr/bin/fc-scan
/usr/bin/fc-validate
```

plus documentation.

Reply:
looks like a typo, this should be `fontconfig libfontconfig1`